azurerm/validators_test.go

```diff
@@ -182,3 +182,37 @@ func TestValidateIntBetweenDivisibleBy(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateAzureVMTimeZone(t *testing.T) {
+	cases := []struct {
+		Value  string
+		Errors int
+	}{
+		{
+			Value:  "UTC",
+			Errors: 0,
+		},
+		{
+			Value:  "China Standard Time",
+			Errors: 0,
+		},
+		{
+			// Valid UTC time zone
+			Value:  "utc-11",
+			Errors: 0,
+		},
+		{
+			// Invalid UTC time zone
+			Value:  "UTC-10",
```

Comment:
Could we get an obviously wrong value like `UTC-42` here?

Reply:
Done.